### DISCUSSION TITLE
Which English Premier League team recently sacked its head coach?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#562] [Galaxy Trivia] [answered]
**Bonus:** How many matches did he stand?

**Comment by peterbaccaro** [comment, marked as answer]
English Premier League teams are sacking managers all the time, within the last month alone the following 4 managers have been sacked:

- Patrick Viera - Crystal Palace - March 17th
- Antonio Conte - Tottenham Hotspur - March 26th
- Brendan Rogers - Leicester - April 2
- Graham Potter - Chelsea - April 2

Brendan Rogers and Graham Potter were the last two managers to be sacked, both on April 2.  Graham Potter's sacking however was announced a few hours after after the announcement that Brendan Rogers had been sacked.

So the answer is **Graham Potter** and he managed **31 matches** for Chelsea before he was sacked.

**Comment by thuongtruong1009** [comment]
the most recent English Premier League team to sack its head coach was Tottenham Hotspur, who parted ways with Jose Mourinho on April 19, 2021. Mourinho stood for 17 matches in the 2020-2021 season before his dismissal. It's important to note that this information may have changed since my knowledge cutoff.
